src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -361,8 +361,18 @@ impl AstDisplay for DbzMode {
 }
 impl_display!(DbzMode);
 
+pub enum ConnectorType {
+    File,
+    Kafka,
+    Kinesis,
+    AvroOcf,
+    S3,
+    Postgres,
+    Pubnub,
+}
+
```

Comment:
Oh, actually, it occurs to me, you should use the `enum_kinds` create here to automatically derive this! There's a few examples in the codebase if you grep around.

Reply:
I tried out `enum_kinds`, but it doesn't feel totally worth it to me! I want a single external `Type` for sources and sinks just to use in `check_statement_safety`. So even with `enum_kinds`, I end up with a `ConnectorType` enum anyway. I think I'll ditch this (in `ddl.rs`) and `enum_kinds` if that makes sense to you!